site/content/contribute/getting-started/core-committers.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Core Committers"
 date: 2017-08-20T12:33:36-04:00
-weight: 2
+weight: 5
```

Comment:
Might actually make sense to have core committers appear before the code review page, since the code review page requires some knowledge of what a core committer is